### PR TITLE
Add Ensemble documentation

### PR DESCRIPTION
**Pull request by nv-braf**
Adding ensemble model support to our documentation. Will file a separate story to add an example ensemble to our documentation when we find a suitable model. I've also removed the `early access` tag from multi-model.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,9 +43,12 @@ Triton Inference Server.
   to test the model with different concurrency and batch sizes of requests. Using
   [Manual Config Search](docs/config_search.md#manual-brute-search), you can create manual sweeps for every parameter that can be specified in the model configuration.
 
-- [Multi-Model Search](docs/config_search.md#multi-model-search-mode): **EARLY ACCESS** - Model Analyzer can help you
+- [Multi-Model Search](docs/config_search.md#multi-model-search-mode): Model Analyzer can help you
   find the optimal settings when profiling multiple concurrent models, utilizing our Quick Search alogrithm
 
+- [Ensemble Model Search](docs/config_search.md#ensemble-model-search): Model Analyzer can help you find the optimal
+  settings when profiling an ensemble model, utilizing our Quick Search algorithm
```
Comment (tgerdesnv) [inline README.md:50]: I don't think we want to say "our". Just say "the"? (Same for multi-model line 47)

File: docs/config_search.md [hunk 2]
```diff
@@ -187,9 +184,19 @@ Using the `--run-config-search-<min/max>...` CLI options you have the ability to
 
 Note: That by default quick search runs unbounded and ignores any default values for these settings
 
+## Ensemble Model Search
```
Comment (tgerdesnv) [inline docs/config_search.md:187]: Should we explicitly say we do not support BLS? Or is that considered different enough from ensembles?

File: docs/config_search.md [hunk 3]
```diff
@@ -187,9 +184,19 @@ Using the `--run-config-search-<min/max>...` CLI options you have the ability to
...
+
+_This mode has the following limitations:_
+
+- Can only be run in `quick` search mode
```
Comment (tgerdesnv) [inline docs/config_search.md:191]: Should we put a max submodel number here? At least as a recommendation?

Comment (nv-braf) [reply docs/config_search.md:191]: Not sure. I mention below that it should take less than one hour for a 4 submodel run, but if they want to run a larger submodel ensemble I don't see why we should restrict them.

Comment (tgerdesnv) [reply docs/config_search.md:191]: I think we should set expectations properly. Let's be upfront that it is only verified or expected to work properly on 4 or 5 and less

**Review by tgerdesnv — APPROVED** [review]

**Pull request merged**